ros/src/waypoint_updater/waypoint_updater.py

```diff
@@ -29,23 +33,64 @@ def __init__(self):
         rospy.init_node('waypoint_updater')
 
         rospy.Subscriber('/current_pose', PoseStamped, self.pose_cb)
-        rospy.Subscriber('/base_waypoints', Lane, self.waypoints_cb)
+        rospy.Subscriber('/base_waypoints', Lane, self.waypoints_cb) #is only published once (latched)
 
         # TODO: Add a subscriber for /traffic_waypoint and /obstacle_waypoint below
 
 
         self.final_waypoints_pub = rospy.Publisher('final_waypoints', Lane, queue_size=1)
 
-        # TODO: Add other member variables you need below
+        self.pose = None
+        self.base_waypoints = None
+        self.waypoints_2d = None
+        self.waypoint_tree = None
+        self.stopline_wp_idx = -1
```

Comment:
do we need this declaration, did not see any usage in file

Reply:
this will be needed later when the traffic detector is implemented.
but you are right it will be removed and re-added later when this functionality is integrated 
and yes it is showing the waypoints in the simulator